scripts/ecs/appsignals/patch-ecs-app.sh

```diff
@@ -0,0 +1,6 @@
+cp application-config/admin-server-application.yml ../../../spring-petclinic-admin-server/src/main/resources/application.yml
```

Comment:
In petclinic, each service registered with eureka with it's IP and port. For example, in customer service, the register configuration is defined in [application.yml file](https://github.com/aws-observability/application-signals-demo/blob/420c8cdbec61ac0b24e05e9c8b41190c0f90dd43/spring-petclinic-customers-service/src/main/resources/application.yml#L7C1-L7C7). 

In current setup, 
1. The default IP is the instance IP address. For example, if the app is running on EC2 instance, eureka will try to find the customer service at EC2 instance IP: `http://<EC2_instance_ip>:<port>`. 
2. If the app is running in Docker file, the app will register it's IP as the container name, for example `customers-service`, and the eureka will try to find the customer service at: `http://customers-service:<port>`

However, in ECS, the app cannot be reachable from either the instance IP (as each task has it's own IP) nor the container name. We use service discovery to expose the app with <`service_name.namespace>` format, for example: `customers-service.ecs-petclinic`. Then the customer service register itself on eureka at `http://customers-service.ecs-petclinic:<port>` In order to do this, we need to override the config file from:
```
eureka:
  instance:
    preferIpAddress: true
```
To:
```
eureka:
  instance:
    preferIpAddress: true
    ipAddress: ${CUSTOMER_SERVICE_IP:customer-service}
```
where `CUSTOMER_SERVICE_IP` is `customers-service.ecs-petclinic`, we pass it through env variable.

This only applies to ECS, since EC2 will continue use it's instance IP, and EKS&Ek8 continue use its container name. Thus, we only apply patch on ECS config setup.

Reply:
Thanks for the explanation. In this case, why not just maintain those property files in the application repository with a different profile name? ref: https://docs.spring.io/spring-boot/docs/1.0.1.RELEASE/reference/html/howto-properties-and-configuration.html

We don't need to copy the files during the setup.

Reply:
Thanks for the suggestion, I wasn't know we can add extra profile. Updated the PR with suggested methods.